src/Autofac.Integration.ServiceFabric/AutofacServiceExtensions.cs

```diff
@@ -53,9 +53,9 @@ public static class AutofacServiceExtensions
         {
             var registration = RegisterServiceWithContainer<TService>(builder, serviceTypeName, lifetimeScopeTag);
 
-            builder.RegisterBuildCallback(c =>
-                c.Resolve<IStatefulServiceFactoryRegistration>()
-                    .RegisterStatefulServiceFactory<TService>(c, serviceTypeName, lifetimeScopeTag));
+            builder.RegisterBuildCallback(l =>
```

Comment:
Is the type of this parameter is still `Action<IContainer>`? If so, no need for the rename.

Reply:
That's one of the changes included in Autofac 5.0 before build callback where only supported at the root scope/container level. Now they are supported at all level so they changed the signature so it takes an Action<ILifetimeScope> instead of an Action<IContainer>.